public/js/draw.js

```diff
@@ -41,6 +51,18 @@ document.addEventListener("DOMContentLoaded", function () {
         socket.on('draw', function(line) {
             draw(line);
         });
+
+        $("form#chat").submit(function (e) {
+            e.preventDefault();
+            var guessWord = $(this).find("#guess_word").val();
+            var name = $(this).find("#user_name").val();
+            socket.emit("send word", { guessWord: guessWord, name: name}, function (cb) {
```

Comment:
It is not secure. Use post and handle it on backend.
If you using that other users will see all guess word sended